compile_single.py

```diff
@@ -89,9 +83,8 @@ def walk_files(main_file, path, package, imports=None):
     main_file.write('package %s;\n\n' % package)
 
     if lang == "go":
-        go_pkg = convert_to_go_package(package)
-        package = "%s/%s" % (go_pkg, go_pkg)
-        main_file.write('option go_package = "%s";\n' % go_pkg)
+        package = go_helper.convert_to_go_package(package)
```

Comment:
Yes, I just wanted the `.pg.go` file names to be reminiscent of idiomatic go as you might want to vendor these files in your project. Package names need to reflect the output `.proto` file, which determines the name of the `.pg.go` file.